inductiva/simulators/seissol.py

```diff
@@ -0,0 +1,58 @@
+"""Class to run SeisSol simulations on a custom container image."""
+
+from typing import List, Optional
+from inductiva import types, tasks, simulators
+
+
+@simulators.simulator.mpi_enabled
+class SeisSol(simulators.Simulator):
+    """Class to run SeisSol simulations on a custom container image."""
+
+    def __init__(self, container_image: str):
+        """
+        Initialize the SeisSol simulator class.
+
+        Args:
+            container_image: The container image to use for the simulation.
+                Example: container_image="docker://inductiva/seissol:latest"
+        """
+        self.container_image = container_image
```

Comment:
No need to change the container_image here. Lets use the docker image we compiled in kutu.

The simulator init will get the docker image based on the class name. In this case, it will look into our kutu repo and search for seissoll docker images and pick the one with the latest version (or if you provide the version it will pick that version if it exists).

Reply:
Take a look into indutiva/simulators/cans.py

Reply:
okay noted